client/src/common/components/MessageBox.jsx

```diff
@@ -15,14 +15,15 @@ const MessageBox = ({ message, type }) => (
       'message-box--gray': type === MessageType.INFO
     })}
   >
-    <span>
+    <span className="message-box__icon">
       <InfoIcon />
     </span>
-    <p className="message-box__content">{message}</p>
+    <p className="message-box__content">{(message, children)}</p>
   </div>
 );
 
 MessageBox.propTypes = {
+  children: PropTypes.node,
```

Comment:
Can this be a string as well?

Reply:
Please see:
https://flow.org/en/docs/react/children/
https://react-cn.github.io/react/tips/children-props-type.html

I would go for `any` 😉

Reply:
But what about this warning??
![Zrzut ekranu 2019-08-2 o 12 09 49](https://user-images.githubusercontent.com/27632432/62362977-76e6e300-b51e-11e9-8e0e-7b4dd79aec74.png)

Reply:
Either disable the rule or reduce the valid prop types to:

```javascript
PropTypes.oneOfType([
  PropTypes.node.isRequired,
  PropTypes.string.isRequired,
  PropTypes.number.isRequired
])
```